packages/node_modules/@ciscospark/spark-core/src/spark-core.js

```diff
@@ -120,6 +122,24 @@ const SparkCore = AmpState.extend({
         });
 
       if (typeof get(attrs, 'credentials.access_token') === 'string') {
+        if (get(attrs, 'credentials.access_token').includes('Bearer') && get(attrs, 'credentials.access_token').split(' ').length - 1 === 0) {
+          this.logger.warn(
+            "Your access token does not have a space between 'Bearer' and the token, please add a space to it or replace it with this already fixed version\n\n",
+            get(attrs, 'credentials.access_token').trim().replace('Bearer', 'Bearer ').replace(/\s+/g, ' ')
+          );
+          this.logger.info("Tip: You don't need to add 'Bearer' to the access_token field, the token by itself is fine");
+
+          set(attrs, 'credentials.access_token', get(attrs, 'credentials.access_token').trim().replace('Bearer', 'Bearer ').replace(/\s+/g, ' '));
+        }
+        else if (get(attrs, 'credentials.access_token').split(' ').length - 1 > 1) {
+          this.logger.warn(
+            `Your access token has one too many spaces (${get(attrs, 'credentials.access_token').split(' ').length - 2}), please clean it up or replace it with this already cleaned version\n\n`,
```

Comment:
- Instead of "one too many", should we just check for more than one and correct it, then say "too many"? 
- ". Please use this format: \n\n"